tests/Testcontainers.Spanner.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1 @@
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Why should we disable parallelization?

Reply:
There is a nasty thing in the implementation for the spanner .net client that requires an environment variable to be set in case of using the emulator: https://cloud.google.com/spanner/docs/emulator#cs 
So when running multiple tests in parrallel, they override eachother environment variable. 
So this environment variable is now also set in the startup, i want to do a little investigation if there is a better place.

Reply:
To be more specific the detection setting in the string will make the client require this environment variable, it will not take the one from the connection string.

Reply:
Oh, that is unfortunate. Is there an upstream issue? It seems weird that we are unable to connect to multiple instances. Is it possible that we could achieve it by configuring a session pool manager (https://github.com/googleapis/google-cloud-dotnet/issues/4970#issuecomment-631376145)?

Reply:
As you can see from the failing tests on ubuntu only, there is a difference in how this is treated in linux and windows, I'll need to gain a more thorough understanding of this before completing this so I'll be trying to find resources/ threads /issues or source code reference on this first.

Reply:
What do you think about utilizing an xUnit.net [collection fixture](https://xunit.net/docs/shared-context) instead of disabling parallelization? This creates a single Cloud Spanner instance for the tests. It would not be necessary to update the environment variable anymore.